stable/prometheus-operator/values.yaml

```diff
@@ -1507,7 +1507,7 @@ prometheus:
     ##
     image:
       repository: quay.io/prometheus/prometheus
-      tag: v2.15.2
+      tag: v2.17.1
```

Comment:
I think we should wait to go to 2.17 until it's a little more stable:
https://github.com/prometheus/prometheus/issues/7120

Reply:
Running 2.17.1 in production since its release without facing this memory issue, but I'm fine with sticking to 2.16.0 for now